pkg/controllers/follower/controller.go

```diff
@@ -290,6 +290,11 @@ func (c *Controller) reconcileLeader(
 			}
 			return worker.StatusError
 		}
+
+		c.metrics.Store("followers_total", len(desiredFollowers),
+			stats.Tag{Name: "namespace", Value: fedObj.GetNamespace()},
+			stats.Tag{Name: "name", Value: fedObj.GetName()},
```

Comment:
```suggestion
			stats.Tag{Name: "name", Value: key.sourceName},
```
should use source object name. /cc @limhawjia

Reply:
this tag has been changed to sourceName of object.